src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -700,6 +700,17 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
 
                     return style.listStyleImage;
                 }
+            },
+
+            // xhr
+            responseURL: {
+                condition: domUtils.isXMLHttpRequest,
```

Comment:
I agree. If we define the `responseURL` property in this place it won't be the cause of any errors and after if we check the `xhr.responseURL` it will be equal to the 'test' string.
But if we append in the test in a `readystatechange` handler the following code:
```javascript
console.log(this.readyState + ' - ' + this.responseURL);
```
it will print the following logs:
```
1 - 
2 - http://localhost:2000/sessionId/https://example.com/xhr-large-response
3 - http://localhost:2000/sessionId/https://example.com/xhr-large-response
4 - http://localhost:2000/sessionId/https://example.com/xhr-large-response
```